assets/js/frontend/checkout.js

```diff
@@ -180,10 +183,30 @@ jQuery( function( $ ) {
 				wc_checkout_form.trigger_update_checkout();
 			}
 		},
+		init_ship_to_different_address: function() {
+			var $checkbox = $( '#ship-to-different-address input' );
+
+			if ( ! $checkbox.prop( 'checked' ) ) {
+				var $billing = $( 'div.woocommerce-billing-fields' );
+
+				var $differentFields = jQuery( 'div.shipping_address' ).find( 'input, select' ).filter( function() {
```

Comment:
Those are inside `jQuery` now it's just fine to use `$`.

Reply:
Whoops – fixed in https://github.com/woocommerce/woocommerce/pull/24271/commits/e15e51868c9b031a473ff7f1a36da94c6a6fabe8 :+1: